database_scripts/sql/004_invitations_and_device_mappings.sql

```diff
@@ -38,7 +38,7 @@ BEGIN
       uid uniqueidentifier NULL,
       createdAt datetime2 NOT NULL,
       updatedAt datetime2 NOT NULL,
-
+      isApprover bit NOT NULL CONSTRAINT [DF_Invitation_isApprover] DEFAULT 0,
```

Comment:
Why do you need this change here?

Reply:
That was a mistake , I have reverted that line of change